### PR TITLE
Sc22 fixes

### PR DESCRIPTION
**Pull request by hdelan**
Some small fixes for SC22 thanks @dkhaldi

### CODE REVIEW

**Comment by hdelan** [comment]
Awkward to cherry pick this into main so will just copy-paste these changes in another PR into main

File: Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html [hunk 1]
```diff
@@ -274,7 +273,7 @@
 {
   <mark>auto hostAcc = buf.get_host_access(cgh);</mark>
```
Comment (dkhaldi) [inline Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html:274]: As I sent you in the exercise corrections email, there is an another issue on this slide:
The definition of hostAcc should be:
               auto hostAcc = bufR.get_host_access();
  or like in the code it is: auto hostAccR = bufR.get_host_access(sycl::read_only);

Comment (hdelan) [reply Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html]: Fixed thanks

File: Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html [hunk 2]
```diff
@@ -222,7 +220,7 @@
 						#### Synchronizing with data
```
Comment (dkhaldi) [inline Lesson_Materials/Lecture_06_Asynchronous_Execution/index.html:220]: I am not able to see how the slides looks like in pdf form.
remember, this particular slide had the text run over the slide.
Was this fixed?

**Review by AerialMantis — APPROVED** [review]
LGTM

**Review by dkhaldi — APPROVED** [review]
LGTM

**Pull request merged**